decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/moderations_controller.rb

```diff
@@ -6,6 +6,21 @@ module Admin
       # This controller allows admins to manage moderations in a participatory process.
       class ModerationsController < Decidim::Admin::ModerationsController
         include Concerns::ParticipatoryProcessAdmin
+        include NeedsPermission
+
+        def permission_scope
+          :admin
+        end
+
+        def permission_class
+          Decidim::ParticipatoryProcesses::Permissions
+        end
+
+        # A convenience method so that spaces can override how to deal with
```

Comment:
Don't understand this. Can you show me an example where that's useful to have?

Reply:
Docs were wrong, it overrides the method from the parent class.